eos-vm-generator

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# Usage: eos-live-boot-generator normal-dir [...]
```

Comment:
copypasta error, should be `eos-vm-generator`

Reply:
Yep, I thought I had changed it but maybe I accidentally reverted it back. Good eye!